src/app/wallet/wallet-tx-history/wallet-tx-history.component.html

```diff
@@ -4,7 +4,7 @@
 
     <button app-action-button
             class="flex items-center justify-center text-white lg:w-auto text-sm rounded-full
-                   bg-indigo-800 hover:bg-indigo-900 py-4 lg:mt-0 lg:px-8 px-4 disabled:bg-gray-400
+                   bg-indigo-800 hover:bg-indigo-900 py-2 lg:mt-0 lg:px-8 px-4 disabled:bg-gray-400
```

Comment:
The buttons on wallet screen have less padding in the design, so I reduced the padding here. Also see `wallet.component.html` lines 26 and 36.
Before:
![Screenshot 2021-09-22 at 17 33 29](https://user-images.githubusercontent.com/12549625/134381759-745eafac-aac8-4f98-9e31-c054f91187b8.png)
After:
![Screenshot 2021-09-22 at 17 36 31](https://user-images.githubusercontent.com/12549625/134381766-df4f03ba-99b0-47a5-affa-1cd9dae1205a.png)